packages/dataviews/src/dataviews-layouts/table/index.tsx

```diff
@@ -98,6 +98,20 @@ function TableColumnField< Item >( {
 	);
 }
 
+function getColumnStyles( view: ViewTableType, columnId: string ) {
```

Comment:
Is there any perf penalty of this regenerating an object on each render? Maybe not